nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-runtime/src/main/java/org/apache/nifi/NiFi.java

```diff
@@ -54,33 +60,38 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Stream;
 
 public class NiFi implements NiFiEntryPoint {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(NiFi.class);
+    public static final String BOOTSTRAP_PORT_PROPERTY = "nifi.bootstrap.listen.port";
+    public static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd_HH-mm-ss");
+
+    private static final Logger logger = LoggerFactory.getLogger(NiFi.class);
```

Comment:
It's not so simple. I check out this last year, because I saw both the lowercase and the uppercase version. [cbliard](https://stackoverflow.com/a/12069687) and [Jetty Breffman](https://stackoverflow.com/a/9510335) wrote two superb comments about this. 
The logger here is not a constant but a constant reference and since it's an interface, a mutable implementation can exist.

Reply:
Currently in NiFi you can find 967 _logger_ and 106 _LOGGER_.